src/ElasticSearch.php

```diff
@@ -252,4 +267,33 @@ public function add_mime_type_filter( $formatted_args ) {
 
 		return $formatted_args;
 	}
+
+	/**
+	 * Debug function to validate/explain a query through elastic server.
```

Comment:
ElasticSearch also has an "[Explain API](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-explain.html)", but it does something else than the "rewrite" parameter we use to get the Lucene query. Maybe we should avoid that term here?